library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -647,6 +746,21 @@ public void highlightDates(Collection<Date> dates) {
     setAdapter(adapter);
   }
 
+  /**
+   * Unhighlights all previously highlighted days.
+   * Also clears {@link #highlightedCells} and {@link #highlightedCals}.
+   */
+  public void unHighlightAllDates() {
```

Comment:
Its different than selected. There is a method `highlightDates(Collection<Date> dates)` from long ago. And `unHighlightAllDates()` removes highlights. `clearHighlightedDates()` though makes sense, will change to that.